packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -112,7 +124,16 @@ class RecoveryPhoneHandler {
         code,
         'signin'
       );
-      return localizedMessage;
+      const shortLocalizedMessage = await this.getLocalizedMessage(
+        request,
+        code,
+        'signin-short'
+      );
+      return {
+        msg: localizedMessage,
+        shortMsg: shortLocalizedMessage,
+        failsafeMsg: `Mozilla: ${code}`,
```

Comment:
Using this failsafe all the time would save a lot of trouble 😅

Reply:
😓